### DISCUSSION TITLE
Try implementing research papers like LangGraph

### DISCUSSION THREAD

**Discussion by HamzaFarhan** [DAGWorks-Inc/burr#251] [Ideas]
Some of the more interesting [LangGraph tutorials](https://langchain-ai.github.io/langgraph/tutorials/usaco/usaco/) show how to implement new and cool research papers. I plan on going through some of them and replacing LangGraph with Burr to get the hang of stuff and build internal tools/utils.
Would that be a nice addition to the examples or maybe a new section?

**Comment by skrawcz** [comment]
Yes please. Would love examples. The only one we've implemented is the collaborative agent one. Please create issues for what you're thinking about and we can assign/go from there?
